backend/hitas/migrations/0025_required_fields.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.15 on 2022-09-28 15:19
```

Comment:
`housing_company_required_fields` would be a bit better name for this file

Reply:
Renamed :)